### PR TITLE
Pool sizes

### PR DESCRIPTION
**Pull request by seanmonstar**
/cc @ckolos

### CODE REVIEW

File: lib/config.js [hunk 1]
```diff
@@ -130,6 +134,10 @@ const conf = convict({
     port: {
       default: '3306',
       env: 'MYSQL_PORT'
+    },
+    connectionLimit: {
```
Comment (vladikoff) [inline lib/config.js]: just browsing through the PRs, I could not find where this `connectionLimit` is used

Comment (seanmonstar) [inline lib/config.js]: This whole `mysql` config object is passed to `mysql.createPool`, so it will be one of the options to the constructor.

Comment (vladikoff) [inline lib/config.js]: Ah okay, both this and `maxSockets` had the same doc `The maximum number of connections that the pool can use at once.` so it was a bit confusing

**Comment by rfk** [comment]
LGTM, r+

**Comment by ckolos** [comment]
Not sure I understand why the verification configs are removed in the browserid.js file. Were they blocking something for the db connections (seems quite unlikely, but wanted to ask anyway....)

**Comment by seanmonstar** [comment]
@ckolos they weren't, the code was just moved around. I created a `request` instance with some values set as default: the `pool` and the `url`, since they never change. The only thing that changes per request is the JSON payload.

**Pull request merged**